packages/terra-infinite-list/tests/wdio/infinite-list-spec.js

```diff
@@ -53,4 +53,17 @@ describe('InfiniteList', () => {
 
     Terra.should.matchScreenshot({ selector: '#test-infinite-list' });
   });
+
+  describe('Displays an infinite list same count and different content', () => {
```

Comment:
Should we have a before and after screenshot to verify the content was actually swapped?